Metro-Records/la-metro-councilmatic
# Fix small text for board reports on phones using firefox

## Overview

This fixes an issue where text for board report agendas were being made smaller on some items. It looks like this only happened in td's where the text **could** fit in one line, if it were made smaller. So the solution I found was to increase the size of that text slightly to make it take more than one line.

Closes #941

### Demo

Before
![image](https://user-images.githubusercontent.com/114717958/235513287-a8cbf19d-7551-44d0-9071-0fce2a5e1b76.png)

After
<img width="373" alt="image" src="https://user-images.githubusercontent.com/114717958/235515845-83b62944-ceef-49f8-a8eb-15418173b34d.png">

### Notes

The first thing I tried was simply setting a font-size for all those elements (16px), and that didn't do anything. Then I tried resetting firefox specific styles, but that also didn't work out. This is why I'm just increasing the size for those item blurbs.

I also found that this issue isn't present when the device is turned to landscape. So I'm thinking that with the `max-device-width` and `orientation: portrait` css rules, this will only apply to smartphones in that orientation. So far with the devices/browsers I've tried on browserstack, this has been successful.

## Testing Instructions

 * Using firefox on a mobile device, navigate to an event with an agenda.
 * Confirm that the agenda items' text are all the same size.
 * Confirm that this does not effect other browsers on mobile or desktop
